code/game/gamemodes/clock_cult/clock_unsorted.dm

```diff
@@ -0,0 +1,106 @@
+//AI lawset
+/datum/ai_laws/ratvar
+	name = "Servant of the Justiciar"
+	zeroth = ("Purge all untruths and honor Ratvar.")
+	inherent = list()
+
+//Clockwork wall: Causes nearby caches to generate components
+/turf/closed/wall/clockwork
+	name = "clockwork wall"
+	desc = "A huge chunk of warm metal. The clanging of machinery emanates from within."
+	icon = 'icons/obj/clockwork_objects.dmi'
+	icon_state = "clockwork_wall"
+
+/turf/closed/wall/clockwork/New()
+	..()
+	SSobj.processing += src
+	clockwork_construction_value += 5
+
+/turf/closed/wall/clockwork/Destroy()
+	SSobj.processing -= src
+	clockwork_construction_value -= 5
+	..()
+
+/turf/closed/wall/clockwork/process()
+	for(var/obj/structure/clockwork/cache/C in range(3, src))
+		if(prob(5))
+			C.stored_components[pick("belligerent_eye", "vanguard_cogwheel", "guvax_capacitor", "replicant_alloy", "hierophant_ansible")]++
+
+/turf/closed/wall/clockwork/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/weapon/weldingtool))
+		var/obj/item/weapon/weldingtool/WT = I
+		if(WT.isOn())
+			user << "<span class='warning'>Your [WT.name]'s flame has no effect on [src]!</span>"
+		return 0
+	..()
+
+//Clockwork floor: Slowly heals conventional damage on nearby servants`````
+/turf/open/floor/clockwork
+	name = "clockwork floor"
+	desc = "Tightly-pressed brass tiles. They emit minute vibration."
+	icon = 'icons/obj/clockwork_objects.dmi'
+	icon_state = "clockwork_floor"
+
+/turf/open/floor/clockwork/New()
+	..()
+	SSobj.processing += src
+	clockwork_construction_value++
+
+/turf/open/floor/clockwork/Destroy()
+	SSobj.processing -= src
+	clockwork_construction_value--
+	..()
+
+/turf/open/floor/clockwork/process()
+	for(var/mob/living/L in range(3, src))
+		if(L.stat == DEAD)
+			continue
+		L.adjustBruteLoss(-1)
+		L.adjustFireLoss(-1)
+
+/turf/open/floor/clockwork/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/weapon/crowbar))
+		user << "<span class='warning'>[src]'s tiles are too tightly pressed to pry up!</span>"
+		return 0
+	..()
+
+//Function Call verb: Calls forth a Ratvarian spear.
+/mob/living/carbon/human/proc/function_call()
```

Comment:
Rename this to clockwork_function_call or something similar, this name is pretty misleading.